### DISCUSSION TITLE
Start myMPD in specified view

### DISCUSSION THREAD

**Discussion by Hajuskin** [jcorporation/myMPD#682] [Ideas]
I think it would be nice to have an option to select in which view myMPD starts. Maybe by adding a checkbox to the tabs context menu:
![mympd](https://user-images.githubusercontent.com/51139914/153729965-4995dd48-96c5-48b1-8ce6-6df84ccacf19.png)

**Comment by jcorporation** [comment]
This should be possible, but you can easily bookmark your preferred view as a browser bookmark.

**Comment by Hajuskin** [comment]
I have it installed as an app in chrome. I guess it is somehow possible to edit the url but especially on the mobile having an option to set the view would by a lot easier.

**Comment by jcorporation** [comment]
Implemented in the v9.2.0 branch. You can select the startup view in the settings dialog.

**Reply by Hajuskin** [reply]
Nice, thank you!
